src/styles/mocto-theme.scss

```diff
@@ -21,6 +21,7 @@ $mocto-disabled-light: #E9E9E9;
 $mocto-disabled: #C4C4C4;
 $mocto-disabled-dark: #7B7B7B;
 
+$mocto-box-height: 19px;
```

Comment:
Building on my comment above, I think we should use spacing tokens. Also is there a specific reason why you used `19px`? We should probably stick to the units used in USWDS. We could use the token `"205"` which corresponds to `20px`.

Reply:
Also [here](https://designsystem.digital.gov/design-tokens/spacing-units/) is the link to the spacing units guide from USWDS. I know it's sometimes difficult to track down the right page in the docs

Reply:
Spacing tokens are used now, didn't need to use `$mocto-box-height` for styling regardless. I knew about the tokens but didn't think we'd need them, thanks!